src/Common/FitOnFhir.Common/Microsoft.Health.FitOnFhir.Common.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Data.Tables" Version="12.5.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.2.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.13.0" />
     <PackageReference Include="Azure.Storage.Queues" Version="12.11.0" />
```

Comment:
Is there a new version of Azure.Storage.Queues that matches the Azure.Storage.Blobs version number 12.13.0? If so we should update it.

Reply:
There is not.  12.11.0 is the latest for Azure.Storage.Queues